Sources/StreamVideoSwiftUI/Livestreaming/LivestreamPlayer.swift

```diff
@@ -9,17 +9,21 @@ import SwiftUI
 public struct LivestreamPlayer: View {
     
     @Injected(\.colors) var colors
-    
+
+    var handleParticipationWithLifecycle: Bool
+    var showsLeaveCallButton: Bool
     var onFullScreenStateChange: ((Bool) -> Void)?
     
     @StateObject var state: CallState
     @StateObject var viewModel: LivestreamPlayerViewModel
-    
+
     public init(
         type: String,
         id: String,
         muted: Bool = false,
         showParticipantCount: Bool = true,
+        handleParticipationWithLifecycle: Bool = true,
```

Comment:
maybe we can simplify this, something like "joinImmediately" (or "joinAutomatically")?